internal/configs/version1/nginx-plus.tmpl

```diff
@@ -39,12 +39,12 @@ http {
     {{if .LogFormat -}}
     log_format  main {{if .LogFormatEscaping}}escape={{ .LogFormatEscaping }} {{end}}
                      {{range $i, $value := .LogFormat -}}
-                     {{with $value}}'{{if $i}} {{end}}{{$value}}'
+                     {{with $value}}'{{if $i}} {{end}}{{$value}} resource_type="$resource_type" resource_name="$resource_name" resource_namespace="$resource_namespace" service="$service"'
                      {{end}}{{end}};
     {{- else -}}
     log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
                       '$status $body_bytes_sent "$http_referer" '
-                      '"$http_user_agent" "$http_x_forwarded_for"';
+                      '"$http_user_agent" "$http_x_forwarded_for" resource_type="$resource_type" resource_name="$resource_name" resource_namespace="$resource_namespace" service="$service"';
```

Comment:
So this would become?:
```suggestion
                      '"$http_user_agent" "$http_x_forwarded_for" '
                      'resource_type="$resource_type" '
                      'resource_name="$resource_name" '
                      'resource_namespace="$resource_namespace" '
                      'service="$service"';
```